### PR TITLE
Refactor & embrace context_info

### PR DESCRIPTION
**Pull request by ghostbuster91**

### CODE REVIEW

File: src/main/java/io/userfeeds/cryptocache/opensea/OpenSeaFacade.kt [hunk 1]
```diff
@@ -33,7 +33,7 @@ class OpenSeaFacade {
                         OpenSeaData(
                                 backgroundColor = it.backgroundColor,
                                 imageUrl = it.imageUrl,
-                                name = it.name,
+                                name = it.name ?: "undefined",
```
Comment (mg6maciej) [inline src/main/java/io/userfeeds/cryptocache/opensea/OpenSeaFacade.kt]: Nope.

File: src/main/java/io/userfeeds/cryptocache/opensea/OpenSeaApi.kt [hunk 2]
```diff
@@ -13,7 +13,7 @@ interface OpenSeaApi {
 
 
     data class OpenSeaAsset(@Json(name = "background_color") val backgroundColor: String?,
```
Comment (mg6maciej) [inline src/main/java/io/userfeeds/cryptocache/opensea/OpenSeaApi.kt]: https://youtu.be/ZsHMHukIlJY?t=18m10s

File: src/main/java/io/userfeeds/cryptocache/opensea/OpenSeaFacade.kt (hunk 1 above)
Comment (ghostbuster91) [reply src/main/java/io/userfeeds/cryptocache/opensea/OpenSeaFacade.kt]: Done.

File: src/main/java/io/userfeeds/cryptocache/opensea/OpenSeaApi.kt (hunk 2 above)
Comment (ghostbuster91) [reply src/main/java/io/userfeeds/cryptocache/opensea/OpenSeaApi.kt]: Done, great movie

**Pull request merged**